src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVFastqUtils.java

```diff
@@ -112,10 +133,16 @@ public FastqRead read( final Kryo kryo, final Input input, final Class<FastqRead
     /** Convert a read's name into a FASTQ record sequence ID */
     public static String readToFastqSeqId( final GATKRead read, final boolean includeMappingLocation ) {
         final String nameSuffix = read.isPaired() ? (read.isFirstOfPair() ? "/1" : "/2") : "";
-        String mapLoc = "";
+        final String mapLoc;
         if ( includeMappingLocation ) {
-            if ( read.isUnmapped() ) mapLoc = " mapping=unmapped";
-            else mapLoc = " mapping=" + read.getContig() + ":" + read.getStart() + ";" + read.getCigar().toString();
+            if ( read.isUnmapped() ) {
+                mapLoc = " mapping=unmapped";
+            } else {
+                mapLoc = String.format(" mapping=%s:%d;%s;strand=%s", read.getContig(), read.getStart(), read.getCigar(),
```

Comment:
I found it rather cleaner this way. considering that most of the time this is associated with slower IO operations I think is better to use format even if it is less efficient. I wonder whether there would be a "pre-compiled" version such as Pattern.compile's but for string composition rather than parsing.